spanner/tests/system/test_system.py

```diff
@@ -394,6 +431,94 @@ def _unit_of_work(transaction, name):
                 COUNTERS_TABLE, COUNTERS_COLUMNS, self.ALL))
         self.assertEqual(len(rows), 2)
 
+    def test_multi_processing_map_transaction(self):
+        from multiprocessing import Pool
+        client = Client()
+        retry = RetryErrors(exceptions.ServiceUnavailable)
+        configs = list(retry(client.list_instance_configs)())
+
+        # Defend against back-end returning configs for regions we aren't
+        # actually allowed to use.
+        configs = [config for config in configs if '-us-' in config.name]
+        if len(configs) < 1:
+            raise ValueError('List instance configs failed in module set up.')
+
+        config_name = configs[0].name
+        instance = client.instance(INSTANCE_ID, config_name)
+        database_name = 'test_database' + unique_resource_id('_')
+        bursty_pool = BurstyPool()
+        database = instance.database(
+            database_name, ddl_statements=DDL_STATEMENTS, pool=bursty_pool)
+        try:
+            operation = database.create()
+            operation.result(30)  # raises on failure / timeout.
```

Comment:
Why not just use the shared database?  Setting one up is slow, and doesn't exercise anything specific to testing multiprocessing (we just need to pass its name through, same as the one generated here).